docs/src/paradox/03-apis/api-v2/tei-xml.md

```diff
@@ -0,0 +1,433 @@
+The Gravsearch template is expected to contain a query involving the text property (URL parameter `textProperty`) and more properties that are going to be mapped to the TEI header. The Gravsearch template is a simple text file with the files extension `.txt`.
+
+A Gravsearch template may look like this (see `webapi/_test_data/test_route/texts/beol/gravsearch.txt`):
+
+```
+PREFIX beol: <http://0.0.0.0:3333/ontology/0801/beol/simple/v2#>
+PREFIX knora-api: <http://api.knora.org/ontology/knora-api/simple/v2#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+    CONSTRUCT {
+        ?letter knora-api:isMainResource true .
+
+        ?letter beol:creationDate ?date .
+
+        ?letter beol:hasText ?text .
+
+        ?letter beol:hasAuthor ?person1 .
+
+        ?person1 beol:hasFamilyName ?name1 .
+
+        ?person1 beol:hasGivenName ?givenName1 .
+
+        ?person1 beol:hasIAFIdentifier ?iaf1 .
+
+        ?letter beol:hasRecipient ?person2 .
+
+        ?person2 beol:hasFamilyName ?name2 .
+
+        ?person2 beol:hasGivenName ?givenName2 .
+
+        ?person2 beol:hasIAFIdentifier ?iaf2 .
+
+
+    } WHERE {
+        BIND(<$resourceIri> as ?letter)
+        ?letter a knora-api:Resource .
+        ?letter a beol:letter .
+
+        ?letter beol:creationDate ?date .
+
+        beol:creationDate knora-api:objectType knora-api:Date .
+        ?date a knora-api:Date .
+
+        ?letter beol:hasText ?text .
+
+        beol:hasText knora-api:objectType xsd:string .
+
+        ?text a xsd:string .
+
+        ?letter beol:hasAuthor ?person1 .
+
+        ?person1 beol:hasFamilyName ?name1 .
+
+        ?person1 beol:hasGivenName ?givenName1 .
+
+        ?person1 beol:hasIAFIdentifier ?iaf1 .
+
+        ?name1 a xsd:string .
+
+        ?givenName1 a xsd:string .
+
+        ?iaf1 a xsd:string .
+
+        ?person2 beol:hasFamilyName ?name2 .
+
+        ?person2 beol:hasGivenName ?givenName2 .
+
+        ?person2 beol:hasIAFIdentifier ?iaf2 .
+
+        ?name2 a xsd:string .
+
+        ?givenName2 a xsd:string .
+
+        ?iaf2 a xsd:string .
+
+        beol:hasGivenName knora-api:objectType xsd:string .
+        beol:hasFamilyName knora-api:objectType xsd:string .
+        beol:hasIAFIdentifier knora-api:objectType xsd:string .
+
+      	beol:hasAuthor knora-api:objectType knora-api:Resource .
+
+        ?letter beol:hasRecipient ?person2 .
+
+      	beol:hasRecipient knora-api:objectType knora-api:Resource .
+
+        ?person1 a knora-api:Resource .
+        ?person2 a knora-api:Resource .
+
+    }
+```
+
+Note the placeholder `BIND(<$resourceIri> as ?letter)` that is going to be replaced by the Iri of the resource the request is performed for.
+The query asks for information about the letter's text `beol:hasText` and information about its author and recipient. This information is converted to the TEI header in the format required by <https://correspsearch.net>.
```

Comment:
Please make this a Markdown link.

Reply:
done in https://github.com/dhlab-basel/Knora/pull/862/commits/670ab38f9277c2bc7874457c93c5e032357bb114